docs/user-guide/plotting.ipynb

```diff
@@ -211,11 +216,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Plotting 2D data\n",
+    "## Plotting 2-D data\n",
     "\n",
-    "### Showing 2D data as an image\n",
+    "### 2-D data as an image\n",
     "\n",
-    "2D variables can be shown as images with a colormap"
+    "2-D variables are plotted as images, with a colormap:"
```

Comment:
I need to add an example where we change the colormap (the name, upper and lower limits, log etc..).
i'll do this in a separate PR